app/models/host/base.rb

```diff
@@ -334,7 +334,7 @@ def set_interface(attributes, name, iface)
       iface.ip = attributes.delete(:ipaddress)
       iface.virtual = attributes.delete(:virtual) || false
       iface.tag = attributes.delete(:tag) || ''
-      iface.attached_to = attributes.delete(:attached_to) || ''
+      iface.attached_to = attributes.delete(:attached_to) if !attributes[:attached_to].nil?
```

Comment:
Nitpick, prefer `if attributes[:attached_to].present?` instead of the negation.  It'll also work then for empty strings as well as nil.